addons/mail/models/ir_websocket.py

```diff
@@ -75,6 +76,24 @@ def _prepare_subscribe_data(self, channels, last):
         data["missed_presences"] = self.env["mail.presence"].sudo().search(presence_domain)
         return data
 
+    def _build_bus_channel_list(self, channels):
+        channels = super()._build_bus_channel_list(channels)
+        chatter_channel_ids = {}
+        regex = re.compile(r"([-_\w\.]*)_(\d+)")
+        for channel in list(channels):
+            if isinstance(channel, str):
+                match = re.match(regex, channel)
+                if match:
+                    model, id = match.groups()
+                    if model in self.env:
```

Comment:
We might want to add a warning when this is not the case. We shouldn't ever send channels for unknown models and if we do it's probably  a mistake we'd like to catch.